orttraining/orttraining/test/python/orttraining_test_ortmodule_experimental_debug.py

```diff
@@ -0,0 +1,63 @@
+
+import os
+import torch
+from onnxruntime.training.ortmodule import ORTModule
+from onnxruntime.capi import _pybind_state as C
+import onnxruntime.training.ortmodule._experimental.debug as debug_config
+
+
+class Net(torch.nn.Module):
+    def __init__(self, input_size=784, hidden_size=500, num_classes=10):
+        super(Net, self).__init__()
+
+        self.fc1 = torch.nn.Linear(input_size, hidden_size)
+        self.relu = torch.nn.ReLU()
+        self.fc2 = torch.nn.Linear(hidden_size, num_classes)
+
+    def forward(self, input1):
+        out = self.fc1(input1)
+        out = self.relu(out)
+        out = self.fc2(out)
+        return out
+
+def test_load_config_from_json():
```

Comment:
Depending on default values to check for changes can be fragile. Default can change and someone could break the loading of that setter, hiding the issue.

Maybe loading 2 different json config would be more accurate, or explicitly setting the config to a value different than the config to make sure defaults are not playing a role

Reply:
I'll add 2 json files for testing.